Software/DumptruckUltra-MCU/hw/pressure_sensor/pressure_sensor.hpp

```diff
@@ -0,0 +1,27 @@
+
+
+#ifndef HW_PRESSURE_SENSOR_HPP_
+#define HW_PRESSURE_SENSOR_HPP_
+
+#include "cyhal_adc.h"
+#include "cyhal_hw_types.h"
+#include <cstdint>
+
+namespace Hardware {
+namespace Pressure_Sensor {
+
+class Pressure {
```

Comment:
Rename to `PressureSensor` for clarity.
```suggestion
class PressureSensor {
```

Reply:
changed in 422f860